cellx/tools/io.py

```diff
@@ -220,11 +222,27 @@ def _test_flagged(_filename: str) -> bool:
     states = {}
 
     # find the zip files:
-    zipfiles = [
-        os.path.join(path, filename)
-        for filename in os.listdir(path)
-        if filename.endswith(".zip") and filename.startswith("annotation_")
-    ]
+    if not isinstance(path, (str, list)):
+        raise IOError(
+            f"Warning, specify correct path format (str or list), not {type(path)}"
+        )
+
+    if isinstance(path, str):
+        zipfiles = [
+            os.path.join(path, filename)
+            for filename in os.listdir(path)
+            if filename.endswith(".zip") and filename.startswith("annotation_")
+        ]
+
+    if isinstance(path, list):
+        zipfiles = [
+            filename
+            for filename in path
+            if filename.split("/")[-1]
+            in os.listdir(("/").join(filename.split("/")[:-1]))
+            and filename.split("/")[-1].startswith("annotation_")
+            and filename.endswith(".zip")
+        ]
```

Comment:
This seems a bit complicated. Perhaps we should just have:
```suggestion
        zipfiles = path
```

and then have a separate check when loading the files? That way, if there is an error, we can skip it and notify the user that the file is missing or filename is malformed?

Reply:
Any better now? Loads the list of files, checks whether in the directory & whether zip file is in the correct format. Prints out if file missing / faulty but continues to the next files in the list.